library/crate

```diff
@@ -7,8 +7,8 @@
 
 # see also https://crate.io
 
-latest: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
+latest: git://github.com/crate/docker-crate@c466b48cf0d243f40ba29eba84c2e8336ebcdeb0
 0.49: git://github.com/crate/docker-crate@8762d54054d9ed9d76a3bba0f9e6fa38acd34c2a
 0.49.6: git://github.com/crate/docker-crate@8762d54054d9ed9d76a3bba0f9e6fa38acd34c2a
 0.50: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
-0.50.2: git://github.com/crate/docker-crate@3a44baa2e8179cb0fdb7540e271b2d2f6f3fa90b
+0.50.3: git://github.com/crate/docker-crate@c466b48cf0d243f40ba29eba84c2e8336ebcdeb0
```

Comment:
Looks like this should've been `5c40f0f6f90d1519f1fda2c6ebe215fdd9d84012` (and `latest` and `0.50` should be too)?